gum/backend-arm/gumstalker-arm.c

```diff
@@ -2715,6 +2732,33 @@ gum_exec_block_obtain (GumExecCtx * ctx,
   return block;
 }
 
+static GumExecBlock *
+gum_exec_block_obtain_trusted (GumExecCtx * ctx,
+                               gpointer real_address,
+                               gpointer * code_address_ptr)
+{
+  GumExecBlock * block;
+  if (ctx->stalker->trust_threshold == 0)
+    return NULL;
```

Comment:
Threshold of zero meant “trust immediately”, so not sure if this check is correct?